src/client/dfuse/il/int_posix.c

```diff
@@ -110,12 +110,12 @@ ioil_shrink_pool(struct ioil_pool *pool)
 }
 
 static int
-ioil_shrink_cont(struct ioil_cont *cont, bool shrink_pool)
+ioil_shrink_cont(struct ioil_cont *cont, bool shrink_pool, bool skip_count_check)
```

Comment:
I think I would prefer this was called force.
```suggestion
ioil_shrink_cont(struct ioil_cont *cont, bool shrink_pool, bool force)
```